i18n/en/design-system.json

```diff
@@ -43,5 +43,25 @@
   "global-footer-licensing-and-vertical-description-param-vertical-music": "Music",
   "global-footer-licensing-and-vertical-description-param-vertical-movies": "Movies",
   "global-footer-full-site-link": "View Full Site",
-  "global-footer-mobile-site-link": "View Mobile Site"
+  "global-footer-mobile-site-link": "View Mobile Site",
+  "global-navigation-fandom-logo-title": "Fandom",
+  "global-navigation-fandom-overview-community-central": "Community Central",
+  "global-navigation-fandom-overview-explore-wikis": "Explore Wikis",
+  "global-navigation-fandom-overview-link-vertical-games": "Games",
+  "global-navigation-fandom-overview-link-vertical-movies": "Movies",
+  "global-navigation-fandom-overview-link-vertical-tv": "TV",
+  "global-navigation-fandom-overview-trending-wikis": "Trending Wikis",
+  "global-navigation-fandom-overview-wikis": "Wikis",
+  "global-navigation-notifications-title": "Notifications",
+  "global-navigation-profile-title": "Profile",
+  "global-navigation-search-placeholder": "Search",
```

Comment:
We use few different strings for placeholders in search. I can't find them here.